tests/components/SearchBar.stories.tsx

```diff
@@ -38,19 +38,6 @@ const meta: ComponentMeta<typeof SearchBar> = {
 export default meta;
 
 export const Primary = (args: SearchBarProps) => {
```

Comment:
can we have Primary.play runs `conductRecentSearches` instead? I believe the other stories would get the recent searches since Primary would set those to local storage beforehand, then we don't need to call `conductRecentSearches` on DropdownExpanded and DropdownHighlight

Reply:
Oh the recent searches setting is needed by those two stories! Alex and I tweaked those a little while back. https://github.com/yext/answers-react-components/commit/1a3dde7da53b9f18f2127bab3bba12d82dc30732

Adding a play(0 function to `Primary` won't work for the stories that need to redefine the play() in order to do additional stuff

Reply:
conducting the recent searches in only one of the stories does set it in local storage for the other stories to use. but, there's no guarantee that the stories run in a particular order (even `Primary` before the others) so it's not dependable to only have it in one